### PR TITLE
Fixes #19401: Add an ldapsearch example in config file for LDAP auth

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/19401

### CODE REVIEW

File: auth-backends/README.adoc [hunk 1]
```diff
@@ -136,12 +136,43 @@ rudder.auth.provider=ldap
...
 # given some base and filter parameters, and then, a bind (rebind) is tried on
 # that entry with the credential provided by the user.
-# That allows to seperate the user DN (especially RDN) from the search criteria.
+# That allows to seperate the user DN (especially RDN) from the search criteria while
+# in the same time supporting users located in several different organisational units.
 #
-# Be careful, the authorization is still done in the rudder-user.xml, what means
-# that each user should have access to Rudder MUST have a line in that file.
+# Be careful, authorizations are still done based on the content of rudder-user.xml,
+# what means that each user should have access to Rudder MUST have a line in that file.
```
Comment (Fdall) [inline auth-backends/README.adoc:143]:
```suggestion
# meaning that each user should have access to Rudder MUST have a line in that file.
```

File: auth-backends/README.adoc [hunk 2]
```diff
@@ -136,12 +136,43 @@ rudder.auth.provider=ldap
...
+#
+# 1/ search for user with `service` login:
+# ----
+# $ ldapsearch -LLL -o ldif-wrap=no -h ldap.mycorp.com -p 389 -x -D "cn=rudder,ou=services,dc=mycorp,dc=com" -w secret -b "ou=Users,dc=mycorp,dc=com" -s sub '(&(cn=jon.doe)(objectclass=person))' 1.1
+#
+#  dn: cn=jon.doe,ou=Paris,ou=Users,dc=mycorp,dc=com
+# ----
+#
+# Errors and unexpected:
+# - an authentication error here means that your rudder service user doesn't havent the
```
Comment (Fdall) [inline auth-backends/README.adoc:161]:
```suggestion
# - an authentication error here means that your rudder service user does not have the
```

File: auth-backends/README.adoc [hunk 3]
```diff
@@ -136,12 +136,43 @@ rudder.auth.provider=ldap
...
+#   rights to do a search and won't be able to find the corresponding user full DN;
```
Comment (Fdall) [inline auth-backends/README.adoc:162]:
```suggestion
#   rights to do a search and will not be able to find the corresponding user full DN;
```

File: auth-backends/README.adoc [hunk 4]
```diff
@@ -136,12 +136,43 @@ rudder.auth.provider=ldap
...
+# - you should get exactly one result: the dn to use in the second request. If you don't
```
Comment (Fdall) [inline auth-backends/README.adoc:163]:
```suggestion
# - you should get exactly one result: the DN to use in the second request. If you don't
```

File: auth-backends/README.adoc [hunk 5]
```diff
@@ -136,12 +136,43 @@ rudder.auth.provider=ldap
...
+#   get any results, check the base DN for search and the LDAP filter.
```
Comment (Fdall) [inline auth-backends/README.adoc:164]:
```suggestion
#   get any results, check the base DN and the LDAP filter.
```

File: auth-backends/README.adoc [hunk 6]
```diff
@@ -151,13 +182,15 @@ rudder.auth.provider=ldap
 rudder.auth.ldap.connection.url=ldap://ldap.mycorp.com:389/dc=mycorp,dc=com
 
 #
-# Bind DN used by Rudder to do the search
+# Bind DN used by Rudder to do the search. This is the "service" or
+# "application" dn for Rudder in you LDAP directory, or an LDAP user with
```
Comment (Fdall) [inline auth-backends/README.adoc:186]:
```suggestion
# "application" DN for Rudder in you LDAP directory, or an LDAP user with
```

**Review by Fdall — CHANGES_REQUESTED** [review]

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**